server/index.ts

```diff
@@ -99,38 +100,18 @@ spectator.generateFirstVotingSequence(level);
 // }, 30000);
 
 io.on("connection", (socket) => {
-  // =====================================================
-  // FIXME: Delete the following lines from the final game.
-  // These are left in for testing convenience.
-  socket.emit("initPlayer", playerCounter)
-  playerCounter += 1
-  playerCounter %= 4
-  // ======================================================
+  if (process.env.VITE_DISABLE_WAITING_ROOM) {
```

Comment:
Let's add this variable to the README table for server env vars in either cases.

(There is a variable on the client side with that same name, but it wasn't on the server's end. While I think it's fine to have just one var used in different places, we could go with another variable if that was preferred?)

Reply:
This is directly related to the variable on the clients end (which is why they have the same name). Is there a way to connect the two together?
We only want automatic playerID assignment when the waiting room is disabled.

Reply:
Actually you are correct. They share the same var when it's deployed, but not locally as it's two processes. I have no complaints otherwise.